src/tests/admin/conferenceroomoutoforderplanning/IfOutOfOrderIsCreatedInThePresentClockIconIsDisplayed.java

```diff
@@ -0,0 +1,59 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC05: Verify that Clock icon appears in out of order column when the schedule to be 
+ * out of order is current time
+ * @author Yesica Acha
+ *
+ */
+public class IfOutOfOrderIsCreatedInThePresentClockIconIsDisplayed {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
+	String roomName = testData.get(3).get("Room Name");
+	String startDate = testData.get(3).get("Start date");
```

Comment:
keep only roomName and title as global variables, others declare inside the test method